--- node_modules/protractor/built/index.d.ts    2020-01-14 09:45:42.123791100 +0200
+++ node_modules/protractor/built/index.d.ts.new        2020-01-14 09:45:54.480272000 +0200
@@ -2,7 +2,6 @@
 import { ElementArrayFinder, ElementFinder } from './element';
 import { ProtractorExpectedConditions } from './expectedConditions';
 import { ProtractorBy } from './locators';
-import { PluginConfig, ProtractorPlugin } from './plugins';
 import { Ptor } from './ptor';
 export { ActionSequence, Browser, Builder, Button, Capabilities, Capability, error, EventEmitter, FileDetector, Key, logging, promise, Session, until, WebDriver, WebElement, WebElementPromise } from 'selenium-webdriver';
 export { ElementHelper, ProtractorBrowser } from './browser';
@@ -10,10 +9,9 @@
 export { ElementArrayFinder, ElementFinder } from './element';
 export { ProtractorExpectedConditions } from './expectedConditions';
 export { Locator, ProtractorBy } from './locators';
+export {PluginConfig, ProtractorPlugin} from './plugins';
 export { Ptor } from './ptor';
 export { Runner } from './runner';
-export declare type PluginConfig = PluginConfig;
-export declare type ProtractorPlugin = ProtractorPlugin;
 export declare let utils: {
     firefox: any;
     http: any;
